script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -43,7 +43,7 @@ Noel-sensei walks over to stand beside me before turning her sights on Arcueid.
 [sha:e011b8e274741862011ed595044980ca71c3a6fa]{
 -- Page 628, Offset 25943.
 -- 10メートル近く吹っ飛ばされていながら傷ひとつない。
-Despite having been blown back nearly ten whole meters, there's not a scratch on her.
+Despite having been blown back nearly 10 whole meters, there's not a scratch on her.
```

Comment:
Volvo is changing everything under 100 to text. This seems to be the more 'correct' approach in EN - 'low' numbers are written out. Exceptions abound.